### DISCUSSION TITLE
Is it possible to disable hot reload for a browser session with a query string?

### DISCUSSION THREAD

**Discussion by axelson** [webpack/webpack#14430] [Q&A]
It seems that at one point appending `?hotreload=false` to your url would disable hot reload:
https://stackoverflow.com/a/54283344

However that doesn't appear to work on Webpack 5 (I'm currently testing on 5.50.0). Is there a way to disable hot reload for an individual browser session without writing any javascript code or having to restart the webpack dev server?

**Comment by alexander-akait** [comment]
Yes, just change it on `hot=false`, here our test https://github.com/webpack/webpack-dev-server/blob/master/test/e2e/hot-and-live-reload.test.js#L263

**Reply by axelson** [reply]
Hmm, that isn't working for me. Although I'm a little behind on webpack-dev-server, I'm on 4.2.1 and latest is 4.3.1. Even when I append `?hot=false` to my url I still have this message in my javascript console `[webpack-dev-server] Hot Module Replacement enabled.`

**Reply by alexander-akait** [reply]
Can you provide steps? We have tests on this
